multi/imxrt-multi/uart.c

```diff
@@ -107,13 +132,66 @@ static inline int uart_getTXcount(uart_t *uart)
 }
 
 
+static int gpio_setup(const uart_halfDuplexAction_t *action)
+{
+	msg_t msg;
```

Comment:
always zero-fill message (either by basic/designated initializer or by memset).

Right now you're sending parts of your stack (possibly leaking secrets) to other process.

Reply:
In this code no messages are actually sent, it directly calls the `gpio_handleMsg` function.

Perhaps to avoid confusion I could make those functions of the GPIO driver part of its public interface.

Reply:
@jmaksymowicz IMHO good idea, would be more efficient than packing/unpacking msg